src/extension.ts

```diff
@@ -25,6 +26,11 @@ export async function activate(context: vscode.ExtensionContext) {
   registerTreeProvider(store, context.extensionPath);
   registerActiveGistTreeProvider(store, context.extensionPath);
 
+  vscode.languages.registerCodeLensProvider(
+    { scheme: "file", language: "markdown" },
```

Comment:
Wouldn’t we want this to be limited to the gist scheme? Since this extension is primarily for gist management?